src/lib/xmltok_impl.rs

```diff
@@ -758,33 +768,14 @@ pub trait XmlTokImpl: XmlEncodingImpl {
                         }
                         return self.scanAtts(buf, nextTokPtr)
                     }
-                    match current_block_161 {
-                        15370445274224965566 => { }
-                        3926109038817298867 => { }
-                        _ => { return XML_TOK::PARTIAL }
-                    }
-                }
-                ByteType::GT => { current_block_161 = 15370445274224965566; }
-                ByteType::SOL => { current_block_161 = 3926109038817298867; }
-                _ => { *nextTokPtr = buf.as_ptr(); return XML_TOK::INVALID }
-            }
-            match current_block_161 {
-                3926109038817298867 => {
-                    buf = buf.inc_start((self.MINBPC()) as isize);
-                    REQUIRE_CHAR!(buf, self);
-                    if !self.char_matches(buf.as_ptr(), ASCII_GT) {
-                        *nextTokPtr = buf.as_ptr();
-                        return XML_TOK::INVALID;
-                    }
-                    *nextTokPtr = buf.inc_start(self.MINBPC()).as_ptr();
-                    return XML_TOK::EMPTY_ELEMENT_NO_ATTS;
+                    return XML_TOK::PARTIAL;
                 }
-                15370445274224965566 => {
+                ByteType::GT => { 
                     *nextTokPtr = buf.inc_start(self.MINBPC()).as_ptr();
                     return XML_TOK::START_TAG_NO_ATTS;
-                }
-                16331546839105579257 => buf = buf.inc_start((self.MINBPC()) as isize),
```

Comment:
Yeah this one looks unreachable, is this the one you meant @thedataking ?

Reply:
indeed!